src/data/sym2.lean

```diff
@@ -282,4 +292,22 @@ equiv_sym α
 
 end sym_equiv
 
+section finite
+
+/--
+Given `[decidable_eq α]` and `[fintype α]`, the following instance gives `fintype (sym2 α)`.
+-/
+instance (α : Type*) [decidable_eq α] : decidable_rel (sym2.rel α) :=
```

Comment:
I'm slightly worried by this. `decidable` is data and this could potentially be used to compute with. We try to avoid defining data in tactic mode because it's not always clear what you're getting. 

The "big guns" here (`cc`) look like they're only used in the `Prop` part, which is fine. But I'd be more comfortable if this were slightly refactored so the algorithmic part was clearer and avoided `rintro` and `subst`.

Reply:
I may have to either leave this for @kmill or just make everything noncomputable for now and wait for Kyle to retrofit it in a separate PR